cluster/restore-from-backup.sh

```diff
@@ -54,8 +54,8 @@ if [ ! -f "${VERSION_FILE}" ]; then
   echo "2.2.1/etcd2" > "${VERSION_FILE}"
 fi
 VERSION_CONTENTS="$(cat ${VERSION_FILE})"
-ETCD_VERSION="$(echo $VERSION_CONTENTS | cut -d '/' -f 1)"
-ETCD_API="$(echo $VERSION_CONTENTS | cut -d '/' -f 2)"
+ETCD_VERSION="$(echo "$VERSION_CONTENTS" | cut -d '/' -f 1)"
```

Comment:
quote to avoid globbing/splitting/ per the comments above this should be a string of the form "binary-version/api-mode" like "2.2.1/etcd2" so we shouldn't have been wanting splitting / globbing here.